source/cloud-security/ms-graph/index.rst

```diff
@@ -22,10 +22,11 @@ Currently, the module allows you to monitor the following with Wazuh:
 - Microsoft Defender for Office 365
 - Microsoft Purview eDiscovery
 - Microsoft Purview Data Loss Prevention (DLP)
+- Microsoft MDM Intune
 
 While these are centric to the security resource, the Microsoft Graph REST API contains a large number of additional resources that can be monitored. See the `Overview of Microsoft Graph <https://learn.microsoft.com/en-us/graph/overview?view=graph-rest-1.0>`_ documentation to learn more.
 
-.. note:: Currently, only the `security` resource can be considered mature as it's the only one tested and with pre-made rules. However, the logs of other resources can still be ingested at your organization's discretion.
+.. note:: Currently, only the `security` and `deviceManagement` resources can be considered mature as they're the only ones tested and with pre-made rules. However, the logs of other resources can still be ingested at your organization's discretion.
```

Comment:
"tested with premade rules" or "tested, and that have premade rules"?

Reply:
Done https://github.com/wazuh/wazuh-documentation/pull/7659/commits/f09284d0aaf9cb2aa1ae216c2b6463a39e9aaa03